Source/ARTCrypto.h

```diff
@@ -44,11 +51,30 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/**
+ The ARTCrypto object ensures that message payloads are encrypted, can never be decrypted by Ably, and can only be decrypted by other clients that share the same secret symmetric key.
```

Comment:
```suggestion
 The ARTCrypto class ensures that message payloads are encrypted, can never be decrypted by Ably, and can only be decrypted by other clients that share the same secret symmetric key.
```

Reply:
I'm not sure whether I agree with this description entirely. I think that the description of how encryption works in Ably is a good one, but it's not the `ARTCrypto` class itself that's responsible for performing the encryption. It’s more of a utility class, which helps you to create the parameters used to configure encryption elsewhere in the SDK.

Reply:
Is something like the following more accurate?

```
The `ARTCrypto` class enables you to set the parameters required for configuring the encryption of message payloads.
```

Reply:
How about

> The `ARTCrypto` class enables you to generate the parameters required for configuring the encryption of message payloads.

@lawrence-forooghian @m-hulbert